src/azure-cli/azure/cli/command_modules/kusto/_help.py

```diff
@@ -10,11 +10,13 @@
 helps['kusto'] = """
 type: group
 short-summary: Manage Azure Kusto resources
+long-summary: This module will not be supported starting January 1, 2021. Please execute "az extension add -n kusto" to install the supported Kusto extension
```

Comment:
You could mark the command as _deprecated_ as well. Which would give user a more intuitive message.

Reply:
Thanks! I've marked the Groups as deprecated.